### PR TITLE
{lib}[goolf/1.7.20] Neggia v1.0.1

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/n/Neggia/Neggia-1.0.1-goolf-1.7.20.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
```
Comment (boegel) [inline easybuild/easyconfigs/n/Neggia/Neggia-1.0.1-goolf-1.7.20.eb]: @pescobar `s/hpcugent/easybuilders/g` to make Travis happy

File: easybuild/easyconfigs/n/Neggia/Neggia-1.0.1-goolf-1.7.20.eb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+name = 'Neggia'
+version = '1.0.1'
+
+homepage = 'https://github.com/dectris/neggia'
+description = "XDS plugin that reads hdf5 files written by Dectris Eiger Detectors"
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['https://github.com/dectris/neggia/archive/']
+sources = ['v%(version)s.tar.gz']
```
Comment (boegel) [inline easybuild/easyconfigs/n/Neggia/Neggia-1.0.1-goolf-1.7.20.eb]: @pescobar Please include a SHA256 checksum

File: easybuild/easyconfigs/n/Neggia/Neggia-1.0.1-goolf-1.7.20.eb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+separate_build_dir = True
+
+builddependencies = [('CMake', '3.9.1')]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/dectris-neggia.so'],
```
Comment (boegel) [inline easybuild/easyconfigs/n/Neggia/Neggia-1.0.1-goolf-1.7.20.eb]: @pescobar Please use `SHLIB_EXT` rather than hardcoding `.so`

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): uses a deprecated toolchain.
For more information on deprecated toolchains, see https://easybuild.readthedocs.io/en/latest/Deprecated-easyconfigs.html#deprecated-toolchains.
Please consider reopening or submitting a new PR using a supported toolchain.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**